From fde6071908d14683d39120daef7d2c0fe3ded0da Mon Sep 17 00:00:00 2001
From: Waldemar Rymarkiewicz <waldemarx.rymarkiewicz@intel.com>
Date: Mon, 16 Jul 2018 14:15:42 +0200
Subject: [PATCH]  PMCPESYS-51: thermal: ltq: Remove unused variables

Moved already to ltq_thermal_sensor, so remove from ltq_thermal.

Signed-off-by: Waldemar Rymarkiewicz <waldemarx.rymarkiewicz@intel.com>
---
 drivers/thermal/ltq_thermal.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/thermal/ltq_thermal.c b/drivers/thermal/ltq_thermal.c
index 8b71f4193057..4d4d167c0fc2 100644
--- a/drivers/thermal/ltq_thermal.c
+++ b/drivers/thermal/ltq_thermal.c
@@ -55,16 +55,9 @@ struct ltq_thermal {
 	struct regmap	*chiptop;
 
 	struct device *dev;
-	struct ltq_thermal_tsens_data *tdata;
-	struct thermal_zone_device *tzd;
 
 	struct ltq_thermal_sensor *sensors;
 	int count;
-
-	int temp;
-	int last_temp;
-
-	int emul_temp;
 };
 
 #define CTRL_REG  0x100
